pkg/pubsub/adapter/converters/pubsub.go

```diff
@@ -48,7 +49,8 @@ func convertPubsub(ctx context.Context, msg *cepubsub.Message, sendMode ModeType
 	// Attributes are extensions.
 	if msg.Attributes != nil && len(msg.Attributes) > 0 {
 		for k, v := range msg.Attributes {
-			event.SetExtension(k, v)
+			// V1 attributes MUST consist of lower-case letters ('a' to 'z') or digits ('0' to '9').
+			event.SetExtension(strings.ToLower(k), v)
```

Comment:
Is there a unit test for this lowercasing?

Reply:
There are no UT for converters, which is even worse...
If we don't lowercase them, then the CE library panics... 
Without this E2E tests don't pass.
If you don't mind, I can create issues to add UTs for converters, and as part of the big refactor we are making, I'll add those...

Reply:
Let me take that back... In order to fix the mistakes from the past... let me actually add them now.. thanks for the comment!

Reply:
Thanks!

Reply:
Added UTs, and changed the logic a bit.